src/pages/[locale]/[...serverPath]/index.astro

```diff
@@ -0,0 +1,36 @@
+---
+import { datocmsCollection } from '@lib/datocms';
+import { isServerRoute } from '@lib/routing/';
+import { getPagePath } from '@lib/routing/page';
+import { PageRoute as fragment, type PageRouteFragment, type SiteLocale } from '@lib/datocms/types';
+import Page from '@views/Page/Page.astro';
+
+export const prerender = false;
+
+export async function getStaticPaths() {
```

Comment:
Are these actually used in this server-rendered route? Or is this now the catch all route in production?